sql/src/test/java/io/crate/metadata/doc/DocIndexMetaDataTest.java

```diff
@@ -1131,8 +1131,8 @@ public void testMergePartitionWithDifferentShardsAndReplicas() throws Exception
                     .endObject()
                 .endObject();
         Settings templateSettings =  ImmutableSettings.builder()
-                .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 1)
-                .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 5)
+                .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 2)
```

Comment:
they were set to the default, so the bug that number_of_shards and number_of_replicas were reset to the default weren't detected, as thevalues of the template should be kept :)